src/constant/constant.js

```diff
@@ -113,6 +113,7 @@ export const PAGE_TITLE = {
     VO_PORTAL: 'Get Parking Spot',
     SO_PORTAL: 'Register Parking Spot',
     TERMS: 'Terms & Conditions',
+    PRIVACY: 'Privacy - Policy',
```

Comment:
Privacy Policy is a single phrase.
We need not add -

Reply:
![privacy-policy-footer](https://github.com/arrogant-ant/parkspot/assets/122720966/baf7c056-65c0-4f3e-8260-d45d18ec8aee)